app/src/main/java/nextstep/payments/ui/newcard/NewCardViewModel.kt

```diff
@@ -0,0 +1,99 @@
+package nextstep.payments.ui.newcard
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import nextstep.payments.data.model.Card
+import nextstep.payments.data.repository.PaymentCardsRepository
+
+data class NewCardUiState(
+    var cardNumber: String = "",
+    var expiredDate: String = "",
+    var password: String = "",
+    var isInitialInput: Boolean = false
+) {
```

Comment:
유효성 검사를 잘 구현해주셨네요 👏 
꼭 가변으로 만들지 않아도 될 것 같아요! 가변으로 구현하면 상태 업데이트를 컴포즈에서 추적하기 어려워집니다.
```suggestion
data class NewCardUiState(
    val cardNumber: String = "",
    val expiredDate: String = "",
    val password: String = "",
    val isInitialInput: Boolean = false
) {
```